src/main/scala/io/sdkman/changelogs/TomcatMigration.scala

```diff
@@ -31,4 +31,31 @@ class TomcatMigration {
       .insert()
     setCandidateDefault("tomcat", "9.0.40")
   }
+
+  @ChangeSet(
+    order = "008",
+    id = "008-update_tomcat_versions",
+    author = "ahmadshalabi"
+  )
+  def migration008(implicit db: MongoDatabase): Document = {
+    removeAllVersions("tomcat")
```

Comment:
Why not keep the legacy versions? SDKMAN is specifically for allowing parallel version of any SDK.

Reply:
Thanks for reminding me to keep the legacy versions.

Reply:
I did the change and added the last 10.1.0 version (10.1.0-M8).